File: password-validation/PasswordValidation.c
#include <stdio.h>
#include <string.h>

int main() {
  char password[80];
  fgets(password, 80, stdin);
  
  int numbers = 0; 
  int symbols = 0; 
  int length = 0;
  
  for (char *p = password; *p; ++p) {
  	 if (*p >= '0' && *p <= '9') ++numbers;
  	 if (strchr("!@#$%&*", *p) != NULL) ++symbols;
  	 ++length;
  }
  
  if (length > 6 && numbers > 1 && symbols > 1) puts("Strong");
  else puts("Weak");
  
  return 0;
}

File: password-validation/PasswordValidation.py
password = input()

length = len(password)
symbols = 0
numbers = 0

if length > 6:
    i = length
    while i > 0:
        i -= 1
        if password[i].isdigit():
            numbers += 1
        elif password[i] in "!@#$%&*":
            symbols += 1

if length > 6 and numbers > 1 and symbols > 1:
    print("Strong")
else:
    print("Weak")
